src/main/java/net/revelc/code/formatter/xml/XMLFormatter.java

```diff
@@ -13,19 +13,54 @@
  */
 package net.revelc.code.formatter.xml;
 
+import java.util.Map;
+
+import net.revelc.code.formatter.AbstractCacheableFormatter;
+import net.revelc.code.formatter.ConfigurationSource;
 import net.revelc.code.formatter.Formatter;
 import net.revelc.code.formatter.LineEnding;
-import net.revelc.code.formatter.jsoup.JsoupBasedFormatter;
 
 /**
  * @author yoshiman
- *
+ * @author jam01
  */
-public class XMLFormatter extends JsoupBasedFormatter implements Formatter {
+public class XMLFormatter extends AbstractCacheableFormatter implements Formatter {
```

Comment:
One thing that could be an issue that I hadn't realized before: the package name here conflicts with the xml-formatter library. In general, this is a bad idea, and could prevent the JVM from loading classes if they are packaged in sealed jars.

Reply:
Hmm was not aware of that. Do you have a change in mind? I suppose the package names should change on the xml-formatter project, since this follows the convention on this project.

Reply:
Yeah, we can change it there.

Reply:
Released the package-renamed version as 0.1.1; should sync to Maven Central within a few hours.